cypress/fixtures/users.json

```diff
@@ -0,0 +1,232 @@
+[
```

Comment:
I think it's better to remove sample e2e tests as soon as they are not related to your project.